opentimelineio/opentime.py

```diff
@@ -568,6 +568,69 @@ def to_timecode(time_obj, rate):
     )
 
 
+def from_time(time_str, rate):
+    """Convert a time with microseconds string into a RationalTime.
+
+    :param time_str: (:class:`str`) A HH:MM:ss.ms time.
+    :param rate: (:class:`float`) The frame-rate to calculate timecode in
+        terms of.
+
+    :return: (:class:`RationalTime`) Instance for the timecode provided.
+    """
+
+    if ';' in time_str:
+        raise ValueError('Drop-Frame timecodes not supported.')
+
+    hours, minutes, seconds = time_str.split(":")
+
+    # Timecode is declared in terms of nominal fps
+    nominal_fps = math.ceil(rate)
```

Comment:
This seems misleading. If I call `from_time("01:00:00.00", 24)` vs `from_time("01:00:00.00", 23.98)` I would expect to get different results. Maybe the conversion to nominal fps should be the responsibility of the caller?

Reply:
Yep, you're right. At first I was coding from the timecode functions, but the ceiling doesn't make sense for absolute time value. I'll figure this out and create a new pull request. Also submitting my contributor licence in the day.
Thanks for the feedbacks